src/main/webapp/search/advanced.xhtml

```diff
@@ -4,216 +4,165 @@
-                                            </div>
-                                        </div>
-                                    </ui:repeat>
-                                    <ui:fragment rendered="#{mdb.id == '1'}">
-                                        <div class="form-group">
-                                            <label class="col-sm-4 control-label">
-                                                <span data-toggle="tooltip" data-placement="auto right" class="tooltiplabel text-info" data-original-title="#{bundle['dataset.metadata.publicationYear.tip']}">
-                                                    #{bundle['dataset.metadata.publicationYear']}
-                                                </span>
-                                            </label>
-                                            <div class="col-sm-6">
-                                                <p:inputText id="dvFieldNamez" styleClass="form-control" value="#{AdvancedSearchPage.dsPublicationDate}"/>
+                                        </label>
+
+                                        <div class="col-sm-6">
+                                            <p:inputText id="stringSearchValue" styleClass="form-control"
+                                                         value="#{item.fieldValue}"
+                                                         rendered="#{item.fieldType == fieldTypeEnums.TEXT or
+                                                          item.fieldType == fieldTypeEnums.TEXTBOX or
+                                                          item.fieldType == fieldTypeEnums.DATE}"/>
+                                            <div>
+                                                <p:inputNumber id="rangeSearchMinimum" style="padding-right: 5px"
+                                                               placeholder="#{bundle['advanced.search.metadata.from']}"
+                                                               value="#{item.minimum}"
+                                                               rendered="#{item.fieldType == fieldTypeEnums.INT or item.fieldType == fieldTypeEnums.FLOAT}"/>
+                                                <p:outputLabel for="rangeSearchMaximum"
+                                                               value="&#8212;"
+                                                               rendered="#{item.fieldType == fieldTypeEnums.INT or item.fieldType == fieldTypeEnums.FLOAT}"/>
+                                                <p:inputNumber id="rangeSearchMaximum" style="padding-left: 5px"
+                                                               placeholder="#{bundle['advanced.search.metadata.to']}"
+                                                               value="#{item.maximum}"
+                                                               rendered="#{item.fieldType == fieldTypeEnums.INT or item.fieldType == fieldTypeEnums.FLOAT}"/>
```

Comment:
I'm thinking if we are able to make this (displaying SearchField) as a reusable component - we have this already defined in three places